src/content/governance/index.md

```diff
@@ -147,13 +147,13 @@ The ability to fork in the face of significant political, philosophical or econo
 
 <Divider />
 
-## Beacon Chain development {#beacon-chain}
+## Beacon Chain governance {#beacon-chain}
 
 The Ethereum governance process often trades off speed and efficiency for openness and inclusivity. In order to accelerate the development of the Beacon Chain, it was launched separately from the proof-of-work Ethereum network and followed its own governance practices.
 
-While the specification and implementations development has always been fully open source, the formal processes used to propose updates described above weren't used. This allowed changes to be specified and agreed upon quicker by researchers and implementers.
+While the specification and development implementations have always been fully open source, the formal processes used to propose updates described above weren't used. This allowed changes to be specified and agreed upon quicker by researchers and implementers.
 
-When the Beacon Chain merges with the Ethereum execution layer, the governance process to propose changes will be harmonized. This process to implement The Merge is [already underway](https://github.com/ethereum/EIPs/pull/3675).
+When the Beacon Chain (consensus layer) merged with the Ethereum execution layer, the governance process to propose changes was harmonized. The Merge was successfully implemented on September 15, 2022 and the governance process that lead to its success can be found [here](https://github.com/ethereum/EIPs/pull/3675).
```

Comment:
Hey :-) I don't think we can say this (yet).

The governance process is still up in the air AFAIA... @jmcook1186?

Reply:
I need to merge dev into this to address conflicts, but https://github.com/ethereum/ethereum-org-website/pull/9165 updated this as well. If we need to revert this I suggest we do it in a separate PR.